{{cookiecutter.repo_name}}/ruff.toml

```diff
@@ -0,0 +1,19 @@
+target-version = "py39"
+
+[lint]
+extend-select = [
+  "UP",  # pyupgrade
+  "D",   # pydocstyle
```

Comment:
I would propose to add the `isort` rule. Also maybe `flake8-bugbear` which might catch actual bugs.
I am not sure about `pydocstyle`, given the number of rules we skip below.

See my config (we definitely do not want all the rules I have, just those that are least painful / have most upside)
https://github.com/ispg-group/aiidalab-ispg/blob/main/pyproject.toml

If we enable isort, we should configure `aiida`, and `aiidalab_widgets_base` as known-first-party packages so that they are grouped together and separately from others, see
https://github.com/ispg-group/aiidalab-ispg/blob/d4ab6ce6972b1be45a6669007cb343c99ecec90c/pyproject.toml#L49